src/pages/write-for-egghead/WritingSubmissionForm.tsx

```diff
@@ -0,0 +1,264 @@
+// import React from 'react'
```

Comment:
two things, the build is failing because this file is effectively empty, should likely just delete it for now

secondly, we use a "kebab case" naming convention so this would be `writing-submission-form.tsx` to conform to that, which isn't a major deal, but nice to be same same in that regard

Reply:
@MaggieAppleton I went ahead and deleted the file that was making the build sad. The alternative to that would be to not comment it out and then also not import it into the mdx